manual/config/seafile-conf.md

```diff
@@ -225,3 +241,30 @@ start time - user id - url - response code - process time
 ```
 
 You can use `SIGUSR1` to trigger log rotation.
+
+## Skip block hash
+
+When a large number of files are uploaded through the web page and API, it will be expensive to calculate the hash value based on the content. Since Seafile-pro-9.0.6, you can add the `skip_block_hash` option to use a uuid to calculate a hash value as a block ID.
```

Comment:
it will be expensive to calculate block IDs based on the block contents. Since Seafile-pro-9.0.6, you can add the `skip_block_hash` option to use a random string as block ID.

同时，这个选项应该移动到 seafile fileserver configurations 这一节里面，不用单独列一节。